index.html

```diff
@@ -18,5 +18,157 @@
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+      <header id="page-title">
+        <h1> A Blog for Gardening</h1>
+          <p class="page_description">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Minima, quidem?</p>
+      </header>
+<!--Header part of the page ends here-->
+
+        <section id="home"> 
+          <img id="logo" src="image/logo.jpeg" alt="logo image of the blog">
+        
+          <nav id="navbar"> 
+              <a href="#home" class="nav-link">Home</a>
+              <a href="#gallery" class="nav-link">Gallery</a>
+              <a href="#articles" class="nav-link">Blog</a>
+          </nav>
+         
+        </section>       
+      
+      <!--main section of the page starts here-->
+      
+      <main class="container">
+        
+        <section id="gallery">    
+                <img class="gallery_image" src="/image/pomegranate-tree.jpg"alt="a photo of a pomegranate tree">
+                <img class="gallery_image" src="/image/big tree.jpg" alt="an image of a big tree in the garden">
+                <img class="gallery_image" src="/image/nature-wallpaper-1.jpeg" alt="picture of a nature wallpaper">
+                <img class="gallery_image" src="/image/garden-1.jpeg" alt="a designed garden's photo">
+                <img class="gallery_image" src="/image/Orange-Persimmons-Growing-on-the-Tree.jpg" alt="a persimmons fruit picture">
+                <img class="gallery_image" src="/image/fruit_bowl.jpg" alt="a photo of a beautiful fruit bowl ">       
```

Comment:
great to see alt text here! You should avoid using phrases like "picture of", "image of" in alt text - accessibility tools can pick up that it is an image because it is contained in an <img /> tag

Reply:
Thank You, for your comment. I changed the alt texts of all the suggested <img> tags.